### PR TITLE
Feature 14/team member profile

### PR DESCRIPTION
**Pull request by jahanner**

### CODE REVIEW

File: web-ui/src/components/profile/Profile.jsx [hunk 1]
```diff
@@ -1,24 +1,27 @@
-import React from "react";
+import React, { useContext } from "react";
+import ProfileContext from "../../context/ProfileContext";
 import "./Profile.css";
 
-const Profile = (profile = {}) => {
-const { image_url } = profile;
+const Profile = () => {
+  const context = useContext(ProfileContext);
+  const { bio, email, image_url, name, PDL, role } = context.defaultProfile;
```
Comment (jorwalk) [inline web-ui/src/components/profile/Profile.jsx:7]: could you please change the variable `PDL` to be consistent with the others?
`const { bio, email, image_url, name, pdl, role } = context.defaultProfile;`

File: web-ui/src/components/team-member/MemberIcon.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  const { profile, onSelect } = props;
+  const { image_url } = profile;
+  let image = image_url ? image_url : "https://i.imgur.com/TkSNOpF.jpg";
+
+  return (
+    <div onClick={() => onSelect(profile)} className="image-div">
+      <img
+        alt="Profile"
+        src={image}
+        style={{ maxWidth: "156px", maxHeight: "156px", marginBottom: "20px" }}
```
Comment (jorwalk) [inline web-ui/src/components/team-member/MemberIcon.jsx:14]: Would it be possible to use a class on the Profile image, and move this inline style declaration to `./TeamMember.css`

File: web-ui/src/components/team-member/TeamMemberContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
+import React, { useContext, useState } from "react";
+import MemberIcon from "./MemberIcon";
+import ProfileContext from "../../context/ProfileContext";
+import "./TeamMember.css";
+
+const TeamMemberContainer = () => {
+  const context = useContext(ProfileContext);
+  // const profiles = context.teamMembers;
```
Comment (jorwalk) [inline web-ui/src/components/team-member/TeamMemberContainer.jsx:8]: can this commend be removed?

File: web-ui/src/components/team-member/TeamMemberContainer.jsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+                  <p>Email: {workEmail}</p>
+                  <p>Bio: {bioText}</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      )}
+      <div className="flex-row" style={{ flexWrap: "wrap" }}>
+        {team.length === 0 ? "No team members :/" : team}
```
Comment (jorwalk) [inline web-ui/src/components/team-member/TeamMemberContainer.jsx:91]: Can we wrap this text with a paragraph tag like:
`{team.length === 0 ? "<p>No team members :/</p>" : team}`

File: web-ui/src/components/team-member/MemberIcon.jsx (hunk 2 above)
Comment (mkimberlin) [reply web-ui/src/components/team-member/MemberIcon.jsx:14]: +1

File: web-ui/src/App.js [hunk 5]
```diff
@@ -1,54 +1,115 @@
...
+
+import "./App.css";
+
+const defaultProfile = {
+  bio: "It was all a dream, I used to read Word Up magazine",
+  email: "Biggie@oci.com",
+  name: "Christopher Wallace",
+  PDL: "Tupac Shakur",
+  role: "Lyrical Poet",
+};
```
Comment (mkimberlin) [inline web-ui/src/App.js:23]: Rollin' with the homies...

Comment (mkimberlin) [reply web-ui/src/App.js:23]: Darn...no image embedding I guess:
https://giphy.com/embed/l0EryJIEpKUc3bYfm

File: web-ui/src/components/team-member/TeamMemberContainer.jsx (hunk 3 above)
Comment (jahanner) [reply web-ui/src/components/team-member/TeamMemberContainer.jsx:8]: That is the context that will be used when we don't have to mock the data anymore

**Pull request merged**